frontend/src/components/site/FooterSocial.module.css

```
@@ -0,0 +1,39 @@
+  place-items: center;
+  border: 1px solid var(--line);
+  border-radius: 999px;
+  color: var(--paper2);
+  background: transparent;
+}
+
+.mark svg {
+  width: 20px;
+  height: 20px;
+  display: block;
+  fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)